#Expyriment 1.0.0 (Python 3.13.7), .xpe-file, coding: cp1252
#date: Tue Oct 07 2025 01:12:11
#sha1: c71683
#modules: 
#display: size=(800, 600), window_mode=True, opengl=2
#os: uname_result(system='Windows', node='Shrimp', release='11', version='10.0.26100', machine='AMD64')
Time,Type,Event,Value,Detail,Detail2
1018,design,log,
#design: #exp: Ternus
#design: block_cnt,block_id,trial_cnt,trial_id
1018,design,logged,
1018,Experiment,started
1037,stderr,received,'Traceback (most recent call last):'
1037,stderr,received,'  File "c:\\Users\\Claire\\progcog\\Assignments\\Week-4\\Exercises\\ternus.py", line 69, in <module>\n    run_trial(True)\n    ~~~~~~~~~^^^^^^'
1038,stderr,received,'  File "c:\\Users\\Claire\\progcog\\Assignments\\Week-4\\Exercises\\ternus.py", line 41, in run_trial\n    load(c2)\n         ^^'
1038,stderr,received,"UnboundLocalError: cannot access local variable 'c2' where it is not associated with a value"
